### DISCUSSION TITLE
Welcome to Institute-template Discussions!

### DISCUSSION THREAD

**Discussion by CodeCraftersgcuf** [CodeCraftersgcuf/Institute-template#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Muhammad-Saim18** [comment]
Hey

**Comment by Muhammad-Saim18** [comment]
I have nothing else to say😂

**Comment by Hamza9466** [comment]
bloge waly page main kia masla araha hai
